library/VTKExtensions/Rendering/vtkF3DInteractorStyle.cxx

```diff
@@ -114,7 +114,19 @@ void vtkF3DInteractorStyle::Dolly()
   {
     return;
   }
-  this->Superclass::Dolly();
+
+  vtkRenderWindowInteractor* rwi = this->Interactor;
+  const double* center = this->CurrentRenderer->GetCenter();
+  const int dy = rwi->GetEventPosition()[1] - rwi->GetLastEventPosition()[1];
+  const int dx = rwi->GetEventPosition()[0] - rwi->GetLastEventPosition()[0];
```

Comment:
doubt it would change anything but, on principle, would it be better to call `GetEventPosition()` and `GetLastEventPosition()` only once each?